p2p/host.go

```diff
@@ -209,31 +322,11 @@ func (h *host) peerCount() int {
 	return len(peers)
 }
 
-// close shuts down the host and its components
-func (h *host) close() error {
```

Comment:
was this removed?

Reply:
Nope just moved [here](https://github.com/ChainSafe/gossamer/pull/427/files#diff-697b58a8ad3efb7d1ebbd09bcf39efb0R123)